nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -110,6 +110,11 @@
 
   protected String errorURL;
 
+  /** --- Fully qualified names of custom nonnull/nullable annotation --- */
+  protected Set<String> customNonnullAnnotation;
```

Comment:
These two should probably also be plural (annotation**s**), as should the comment above.

Reply:
441dd40ea4d83e8b36e341fffc785b60a5e3b0ad